system-parachains/constants/src/kusama.rs

```diff
@@ -23,6 +23,16 @@ pub mod consensus {
 	pub const BLOCK_PROCESSING_VELOCITY: u32 = 1;
 	/// Relay chain slot duration, in milliseconds.
 	pub const RELAY_CHAIN_SLOT_DURATION_MILLIS: u32 = 6000;
+
+	/// Parameters enabling async backing functionality.
+	///
+	/// Once all system chains have migrated to the new async backing mechanism, the parameters
+	/// in this namespace will replace those currently defined in `super::*`.
+	pub mod fast {
```

Comment:
We could just call it `async_backing` or something, at least would be more clear to me.

Reply:
renamed